### PR TITLE
APMSubFrameComponent: Add confirmation dialog

### PR DESCRIPTION
**Pull request by patrickelectric**
https://user-images.githubusercontent.com/1215497/127672323-fb5111fa-9592-4a36-9558-ac8595870527.mp4

### CODE REVIEW

File: src/AutoPilotPlugins/APM/APMSubFrameComponent.qml [hunk 1]
```diff
@@ -25,9 +25,20 @@ SetupPage {
...
 
     APMAirframeComponentController { id: controller; }
 
+    property Fact _frameConfig: controller.getParameterFact(-1, "FRAME_CONFIG")
+
+    function setFrameConfig(frameNumber) {
+        _frameConfig.value = frameNumber
+    }
+
+    function setFrameParameter(frameNumber) {
```
Comment (Williangalvani) [inline src/AutoPilotPlugins/APM/APMSubFrameComponent.qml:38]: This function name sounds off. What about `loadFrameDefaultParameters`?

File: src/AutoPilotPlugins/APM/APMSubFrameComponent.qml [hunk 2]
```diff
@@ -110,12 +103,14 @@ SetupPage {
                     name: "BlueROV2/Vectored"
                     resource: "qrc:///qmlimages/Frames/Vectored.png"
                     paramValue: 1
+                    paramName: "bluerov2"
```
Comment (Williangalvani) [inline src/AutoPilotPlugins/APM/APMSubFrameComponent.qml:106]: `paramFileName` maybe?

**Review by Williangalvani — CHANGES_REQUESTED** [review]
looks good, I just found a couple of the names confusing.

File: src/AutoPilotPlugins/APM/APMSubFrameComponent.qml (hunk 1 above)
Comment (patrickelectric) [reply src/AutoPilotPlugins/APM/APMSubFrameComponent.qml]: Done.

File: src/AutoPilotPlugins/APM/APMSubFrameComponent.qml (hunk 2 above)
Comment (patrickelectric) [reply src/AutoPilotPlugins/APM/APMSubFrameComponent.qml]: Done.

**Pull request merged**